kv_cache_calculator.html

```diff
@@ -95,11 +102,14 @@ <h1>KV Cache Size Calculator</h1>
         <div id="calculation-steps"></div>
         <button id="githubButton" onclick="openGitHubRepo()">Contribute new models on GitHub</button>
     </div>
+    <footer>
+        Developed by Zhuohan Gu @ LMCache team
+    </footer>
     <script>
         let modelConfigs = {};
 
         function openGitHubRepo() {
-            const githubUrl = 'https://github.com/LMCache/LMCache/tree/dev/examples/kv_cache_calculator';
+            const githubUrl = 'https://github.com/LMCache/LMCache/issues/244#:~:text=https%3A//github.com/LMCache/LMCache/tree/dev/examples/kv_cache_calculator';
```

Comment:
This link is wrong, it should be just be `https://github.com/LMCache/LMCache/tree/dev/examples/kv_cache_calculator`